rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -573,7 +582,6 @@ rm -f %{_builddir}/file.list.%{name}
 
 %if "%{?aix}" == ""
 # no init no cron and no profile with aix
-%config /etc/cron.d/rudder-agent
```

Comment:
don't we need to add it as normal file?

Reply:
It is already declared as a regular %file, otherwise %config would produce an error